procedural/main.cpp

```diff
@@ -155,6 +155,11 @@ procedural_viewport
 
     applyProceduralSearchPath(filename, universe);
 
+    if (!UsdStage::IsSupportedFile(filename)) {
+        AiMsgError("[usd] Filename not supported : %s", filename.c_str());
```

Comment:
I think `File not supported` sounds better, that also matches the USD function `UsdStage::IsSupportedFile`.